### PR TITLE
Update connect-jupyter-notebooks.md

### PR DESCRIPTION
**Pull request by originou**
Add documentation how-to add dependencies for scala notebook

# Jira Ticket
[BGD-2545](https://spotinst.atlassian.net/browse/BGD-2545)

### CODE REVIEW

File: src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md [hunk 1]
```diff
@@ -92,6 +92,17 @@ in your configuration template. Here's an example configuration for a Scala kern
...
+- `deps.jars` field on the Configuration template does not work with Scala Notebook and **should not be set**
+- Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):
+  - Add a JAR with URL: `%AddJar <URL>`
+      - example: `%AddJar https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.20/postgresql-42.2.20.jar`
+  - Add a dependency from maven repo: `%AddDeps <group-id> <artifact-id> <version>`
+    - example: `%AddDeps org.postgresql postgresql 42.2.20`
+    - if the dependency has transitive dependencies, you can add the `--transitive` flag to add those dependencies  
+
+More documentation for these [magic commands](https://toree.incubator.apache.org/docs/current/user/faq/): 
+
```
Comment (jrj-d) [inline src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md:105]:
```suggestion
**Warning**: Adding external jar dependencies to Scala Notebooks  
The `deps.jars` field in the application configuration does not work with Scala Notebooks and **should not be set**. The JARs specified in this field will not be available on the driver Java classpath.

Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):
- Add a JAR with URL: `%AddJar <URL>`
```
%AddJar https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.20/postgresql-42.2.20.jar
```
- Add a dependency from maven repo: `%AddDeps <group-id> <artifact-id> <version>`
```
%AddDeps org.postgresql postgresql 42.2.20
```
If the dependency has transitive dependencies, you can add the `--transitive` flag to add those dependencies.  

More documentation for these magic commands is available in the [Toree documentation](https://toree.incubator.apache.org/docs/current/user/faq/)

```

Comment (jrj-d) [reply src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md:105]: GitHub messes up my comment, I'll send it to you on Slack

File: src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md [hunk 2]
```diff
@@ -92,6 +92,18 @@ in your configuration template. Here's an example configuration for a Scala kern
  }
 ```
 
+**Warning**: Adding external jar dependencies to Scala Notebooks  
+The `deps.jars` field in the application configuration does not work with Scala Notebooks and **should not be set**. The JARs specified in this field will not be available on the driver Java classpath.
+
+Instead, you can add external jar to spark context from the notebook with these magic commands (once the spark session is up):
```
Comment (jrj-d) [inline src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md:98]:
```suggestion
Instead, you can add an external JAR to the Spark context from the notebook itself with these magic commands (once the spark session is up):
```

File: src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md [hunk 3]
```diff
@@ -92,6 +92,18 @@ in your configuration template. Here's an example configuration for a Scala kern
  }
 ```
 
+**Warning**: Adding external jar dependencies to Scala Notebooks
```
Comment (jrj-d) [inline src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md:95]:
```suggestion
**Warning**: Adding external JAR dependencies to Scala Notebooks  
```

Comment (originou) [reply src/docs/ocean-spark/tools-integrations/connect-jupyter-notebooks.md]: 👍

**Review by jrj-d — APPROVED** [review]
Thanks Denis! Great workaround

**Pull request merged**